packages/typespec-test/test/ai/generated/typespec-ts/review/ai-client.api.md

```diff
@@ -338,9 +338,6 @@ export interface AgentsSubmitToolOutputsToRunOptionalParams extends OperationOpt
     stream?: boolean | null;
 }
 
-// @public
-export type AgentStreamEvent = string | (ThreadStreamEvent | RunStreamEvent | RunStepStreamEvent | MessageStreamEvent | ErrorEvent | DoneEvent);
```

Comment:
These models are removed @tadelesh and here is the link: https://github.com/Azure/autorest.typescript/tree/7923b6b4915b1f1c797efe0ecbdb1281b5c19c5b/packages/typespec-test/test/ai/spec.